### DISCUSSION TITLE
[H5] 不完全支持所有标签吗？

### DISCUSSION THREAD

**Discussion by abitwhy** [NervJS/taro#13427] [Q & A]
h5 项目中发现标签 `thead`、`tfoot`、`tbody` 会导致编译失败。是否 h5 编译模式本就不支持全部标签？
<details>
  <summary>报错</summary>
  <pre>
  [WXML Runtime warning] ./base.wxml
   Template `tmpl_0_thead` not found.
  </pre>
</details>

并没有在[文档](https://taro-docs.jd.com/docs/h5)或[相关 RFC](https://github.com/NervJS/taro-rfcs/blob/html-support/rfcs/0004-rendering-html.md#%E4%B8%80%E6%A0%87%E7%AD%BE%E5%90%8D%E5%BD%B1%E5%B0%84) 中找到相关声明。

**Comment by ZakaryCode** [comment]
你这编译的是微信小程序啊，并不是 H5
